lib/src/util/customPropTypes.ts

```diff
@@ -49,3 +49,21 @@ export const optionalPercentNumber: PropTypes.Validator<number> =
     }
     return requiredPercentNumberValidator(props, propName, componentName);
   };
+
+export const requiredValueNumber: PropTypes.Validator<number> =
```

Comment:
I think this is a little too specific for this file: it relies on there being another prop called `maxVal` which is specifically a donut chart thing.  We don't generally worry that much about custom prop types validators, I'd just use `PropTypes.number` for this the new prop.

Reply:
This is removed ddb1cf189874e4a07b3cdbb84a00febddd6e2e1d